src/components/MultiStepProposalForm.tsx

```diff
@@ -0,0 +1,78 @@
+import {
+  useRef,
+  useImperativeHandle,
+  forwardRef,
+  FormEvent,
+  ReactNode,
+  useState,
+  useMemo,
+} from "react";
+import { Button } from "./Button";
+import { Stepper } from "./Stepper";
+
+interface ProposalFormProps {
+  title: string;
+  description: string | ReactNode;
+  tabs: { title: string; content: ReactNode }[];
+  handleSubmit: (e: FormEvent) => Promise<void>;
+  titleDescOnly?: boolean;
+}
+
+interface ProposalFormMethods {
+  reset: () => void;
+}
+
+const MultiStepProposalForm = forwardRef<
```

Comment:
The black line around the selected tab is kinda weird looking.

![image](https://github.com/DCFoundation/cosmos-proposal-builder/assets/150986/3785a0e1-dce8-410c-9940-b146927bb8e6)

Reply:
Is this always the case, or only with keyboard nav? If the latter, I can make the focus outline more subtle.

Reply:
I used my mouse to select the Step 2 tab

Reply:
[f31d314](https://github.com/DCFoundation/cosmos-proposal-builder/pull/30/commits/f31d31480de53ca929177bb47762ab0739d8870a)